support/models.py

```diff
@@ -34,5 +38,20 @@ class ResourceTypeChoices(models.TextChoices):
     attachment = models.FileField(null=True, blank=True, default=None)
     tags = models.ManyToManyField("ResourceTag", related_name="resources")
 
+    def __str__(self):
+        return self.title
+
     class Meta(OrderedModel.Meta):
         pass
+
+
+class Action(UserStampedModel, TimeStampedModel, OrderedModel):
+    title = models.CharField(max_length=255)
+    organization = models.CharField(max_length=255)
+    summary = models.TextField()
+    description = RichTextUploadingField()
+    latitude = models.DecimalField(max_digits=10, decimal_places=8)
```

Comment:
pointfield should be better

Reply:
https://github.com/openwisp/django-rest-framework-gis

This can come in handy in future to standardize geofields in API